account_payment_term_security/models/account_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import json
+
+from lxml import etree
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def fields_view_get(
```

Comment:
Why do it dynamically instead of putting 2 times the same field, one with readonly and the other without, with the corresponding groups?

Reply:
I have preferred to make this solution to avoid:
- Having to define the new field with what I had previously defined and other modules could add (groups, readonly, attrs, etc).
- If another module wants to use the field selector to change something it will be a problem because it will have to do it twice (the existing field and the new one).

Anyway, if you think it's better to add the field in the view and have it appear twice, no problem.

Reply:
OK, I see. Let's keep it this way.